src/pages/AppDetails/AppDescription.tsx

```diff
@@ -0,0 +1,68 @@
+import * as React from 'react';
+import { App } from '../../types/App';
+import { Badge, Card, CardBody, CardHeader, Title, Tooltip, TooltipPosition } from '@patternfly/react-core';
+import DetailDescription from '../../components/Details/DetailDescription';
+import { serverConfig } from '../../config';
+import Labels from '../../components/Label/Labels';
+import { style } from 'typestyle';
+import * as H from '../../types/Health';
+import { DisplayMode, HealthIndicator } from '../../components/Health/HealthIndicator';
+
+type AppDescriptionProps = {
+  app?: App;
+  health?: H.Health;
+};
+
+const iconStyle = style({
+  margin: '0 0 0 0',
+  padding: '0 0 0 0',
+  display: 'inline-block',
+  verticalAlign: '2px !important'
+});
+
+const healthIconStyle = style({
+  marginLeft: '10px',
+  verticalAlign: '-1px !important'
+});
+
+class AppDescription extends React.Component<AppDescriptionProps> {
+  render() {
+    const appLabels: { [key: string]: string } = {};
+    if (this.props.app) {
+      appLabels[serverConfig.istioLabels.appLabelName] = this.props.app.name;
+    }
+    return this.props.app ? (
+      <Card id={'AppDescriptionCard'}>
+        <CardHeader>
+          <Title headingLevel="h5" size="lg">
+            <div key="service-icon" className={iconStyle}>
+              <Tooltip position={TooltipPosition.top} content={<>Application</>}>
+                <Badge className={'virtualitem_badge_definition'}>A</Badge>
+              </Tooltip>
```

Comment:
Please replace this with the new PFBadge:

`<PFBadge badge={PFBadges.App} position={TooltipPosition.top} />`